### DISCUSSION TITLE
I18n `title` in astro config

### DISCUSSION THREAD

**Discussion by mrzachnugent** [withastro/starlight#989] [Feature Requests]
### What version of `starlight` are you using?

0.11.1

### What is your idea?

In the astro config, it would be nice to have I18n for the `title`

### Something like

```js
starlight({ title: { en: "English" , fr: "Français" } })
```

### Why is this feature necessary?

This feature would help with internationalization right out of the box.

### Do you have examples of this feature in other projects?

```js
starlight({ title: { en: "English" , fr: "Français" } })
```

### Participation

- [ ] I am willing to submit a pull request for this feature.

**Comment by HiDeoo** [comment]
Starlight [`0.22.0`](https://github.com/withastro/starlight/releases/tag/%40astrojs%2Fstarlight%400.22.0) added support for translating the site title.

Documentation for this feature is available [here](https://starlight.astro.build/guides/i18n/#translate-the-site-title) but the change would be:

```diff
import { defineConfig } from 'astro/config';
import starlight from '@astrojs/starlight';

export default defineConfig({
  integrations: [
    starlight({
-      title: 'My Docs',
+      title: {
+         en: 'My Docs',
+        'zh-CN': '我的文档',
+     },
      defaultLocale: 'en',
      locales: {
        en: { label: 'English' },
        'zh-cn': { label: '简体中文', lang: 'zh-CN' },
      },
    }),
  ],
});
```
